packages/wonder-blocks-tooltip/src/components/tooltip.tsx

```diff
@@ -209,7 +212,8 @@ export default class Tooltip extends React.Component<Props, State> {
                 {(props) => (
                     <TooltipBubble
                         id={bubbleId}
-                        style={props.style}
+                        // Allow additional styling to be applied
+                        style={[props.style, style]}
```

Comment:
**question:** Do we have a business need for this? If not, I don't think we should make this change. If we do, then I think we need to discuss the right way to provide it.

I am concerned about styling the bubble directly this way as it creates a competition between the calculated styles from the popper that govern positioning and such, and the ones passed in. If we do want to provide these custom styles, I think they need to be provided in a way that doesn't compete this way, otherwise I think we're potentially inviting a really easy way to get buggy tooltips.

I think this is worth discussing more when @jandrade is back next week.

Reply:
Good question @somewhatabstract. This was needed for supporting some custom backgrounds in Perseus tooltips.

You can see some context here: https://github.com/Khan/perseus/pull/285

I agree with your concerns about allowing to override certain styles passed by Popper, so maybe we can find a way to pass custom styles using a different prop, or applying it to a different container inside the bubble.

Reply:
Could the `TooltipContent` component take the prop instead?

Reply:
i modified it so that we can only pass a subset of styles (`color` of the text, `padding`, and `backgroundColor`). this keeps the scope of the styling small but still covers the original intent of this ticket.